### PR TITLE
Use twig namespaces

### PR DESCRIPTION
**Pull request by nstriedinger**
Refractor styleguide for other systems.

### CODE REVIEW

**Comment by lukewertz** [comment]
@legostud, Nate put up this PR that should provide the refactor we need to leverage these assets directly in Drupal.

Would you have time today to review this? I'd be happy to hop on a call with you and look at it together.

File: styleguide/source/_patterns/01-atoms/01-buttons/button-search.twig [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <button type="submit" class="ma__button-search">
   <span>Search</span>
   <svg role="img" class="ma__button-search__icon">
-    <use xlink:href="/assets/images/svg-sprite.svg#search"></use>
+    <use xlink:href="/{{ directory }}/images/svg-sprite.svg#search"></use>
   </svg>
```
Comment (gender404) [inline styleguide/source/_patterns/01-atoms/01-buttons/button-search.twig:5]: should be {{directory}} not {{ directory }}

File: styleguide/source/_data/data.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
   "title" : "Mass Gov",
   "htmlClass": "pl",
   "bodyClass": "",
+  "directory": "assets",
```
Comment (gender404) [inline styleguide/source/_data/data.json:6]: I know this is a lot to change, but can we use "assetsDir" instead of "directory" to make it more readable

**Review by gender404 — COMMENTED** [review]
1. Is this a WIP request or is it ready to be merged (after review changes)?
2. If you open "show pattern info" (gear menu) does the panel still show patterns it contains and where this pattern is included?
3. Have you searched for {% include path with data %} versions (mainly for Rich Text)?

File: styleguide/source/_patterns/01-atoms/01-buttons/button-search.twig (hunk 1 above)
Comment (nstriedinger) [reply styleguide/source/_patterns/01-atoms/01-buttons/button-search.twig:5]: Our linters use {{ directory }}, but I can change this and this corroborates http://twig.sensiolabs.org/doc/coding_standards.html

Comment (gender404) [reply styleguide/source/_patterns/01-atoms/01-buttons/button-search.twig:5]: hmm, if {{ directory }} is the best practice then we can leave it that way, but you'll need to update all of the other files instead which have {{directory}}.  I'll start using that best practice as well on files as I build them.

File: styleguide/source/_data/data.json (hunk 2 above)
Comment (nstriedinger) [reply styleguide/source/_data/data.json:6]: This just makes it easier for us and builds on our system theme paths. I would have to refractor system paths (Drupal currently), if we change this. If this works and is somewhat arbitrary, lets keep this for the current said system.

File: styleguide/source/_patterns/01-atoms/01-buttons/button-search.twig (hunk 1 above)
Comment (nstriedinger) [reply styleguide/source/_patterns/01-atoms/01-buttons/button-search.twig:5]: 1. Ready to be merged
2. This looks to be right.
3. Yes, but this was a massive refractor. I think things are in place.
4. Please review.

**Comment by gender404** [comment]
Looks like we've disabled one of Pattern Labs features:

image

**Comment by nstriedinger** [comment]
I think this is a bug in PatternLabs. I've reported the issue.

**Pull request merged**